### PR TITLE
Chore: Rework documentation layout using mkdocs

### PR DESCRIPTION
**Pull request by steven-noorbergen**
This PR will likely get a few more updates before merging. Discussions are had on discord ( https://eveonline.com/discord ) in the #3rd-party-dev-and-esi channel.

Contains additions from @kennethjor based on some of my initial work testing out this layout.

Before people ask, yes, I cut out almost all existing documentation, because most documentation is >5 years old, so it's easier to cherry-pick all the relevant information once this is merged.

### CODE REVIEW

File: yaml.schemas [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  "yaml.customTags": [ 
+    "!ENV scalar",
+    "!ENV sequence",
+    "!relative scalar",
+    "tag:yaml.org,2002:python/name:material.extensions.emoji.to_svg",
+    "tag:yaml.org,2002:python/name:material.extensions.emoji.twemoji",
+    "tag:yaml.org,2002:python/name:pymdownx.superfences.fence_code_format",
+    "tag:yaml.org,2002:python/object/apply:pymdownx.slugs.slugify mapping"
+  ]
+}
```
Comment (TrueBrain) [inline yaml.schemas:14]: I am just repeating what GitHub says: no newline is bad :)

File: docs/services/image-server/index.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+Valid values for the `size` parameter include powers-of-two ranging between 32 and 1024.
+If this parameter is not provided, the server will return the image in its original size.
+
+The `tenant` parameter defines the server for which to request images. This defaults to `tranquility`, but also accepts `singularity`.
+
+### Notes
+
+- NPC faction logos are available in the `corporations` category using their faction ID.
+- The ID 1 can be used with alliances, corporations and characters to retrieve the default logo/portrait.
+- Images are returned as PNGs, with the exception of character portraiuts, which are returned as JPEGs.
```
Comment (Dusty-Meg) [inline docs/services/image-server/index.md:33]: Portraits

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  "[markdown]": {
+    "editor.tabSize": 4
+  },
+  "[yaml]": {
+    "editor.tabSize": 2
+  },
+  "[github-actions-workflow]": {
+    "editor.tabSize": 2
+  },
+  // Tabs should be spaces.
```
Comment (ppfeufer) [inline .vscode/settings.json:49]: JSON does not support comments.

Comment (steven-noorbergen) [reply .vscode/settings.json:49]: VSCode config files use JSONC (JSON with Comments) instead of JSON

File: .gitignore [hunk 4]
```diff
@@ -1,2 +1,4 @@
-.venv
-__pycache__
+.env
+.venv/
+/site/
+/.cache/
```
Comment (kennethjor) [inline .gitignore:4]: I'll be adding this later anyway
```suggestion
/.cache/
/.idea/
```

**Review by kennethjor — CHANGES_REQUESTED** [review]

**Pull request merged**